coverage/clover.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Coverage information should not be stored in repository - please add to gitignore.